torch/csrc/jit/codegen/cuda/runtime/random_numbers.cu

```diff
@@ -46,7 +46,7 @@ __device__ uint4 philox(
 
 __device__ float uniformf(unsigned int x) {
   constexpr float kRanInvM32 = 2.3283064e-10f; // Inverse of 2^32.
-  float result = x * kRanInvM32;
+  float result = x * kRanInvM32 + kRanInvM32 / 2.0f;
```

Comment:
This is to ensure the implementation is in consistent with curand. The extra term is there in cuda 11.6 curand version.

The box-muller normal RNG uses uniform RNG first from this function. Without this extra term, the normal RNG would have large numerical differences from curand results.

Reply:
Does the uniform tests pass on both pre-11.6 and post-11.6?

Reply:
The tests passed locally on my cuda 11.6 environment. I'll start a build & test on cuda 11.8 environment as well.

Reply:
I've checked that the new tests for Normal and Uniform pass on cuda 11.8 A100 as well. 
```
build/bin/test_jit --gtest_filter='*FusionNormal*:*FusionUniform*'
```